magento/pwa-studio
# Changes accordion section buttons to type button

## Description

`button` elements that do not submit forms should have `type="button"`. 
This PR fixes the `Accordion` `Section` buttons, which did not.

## Related Issue





Closes PWA-524.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the `/cart` or `/checkout` page where an `Accordion` is rendered (the `Price Adjustments` section)
1. Inspect element
1. See that the section buttons have `type="button"`

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
